staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/client.go

```diff
@@ -39,7 +39,7 @@ var (
 	ErrNeedServiceOrURL = errors.New("webhook configuration must have either service or URL")
 )
 
-// ClientManager builds REST clients to talk to webhooks. It caches the clients
+// clientManager builds REST clients to talk to webhooks. It caches the clients
 // to avoid duplicate creation.
 type ClientManager struct {
```

Comment:
clientManager

Reply:
it's used outside of the package

Reply:
Then the annotation should be ClientManager

Reply:
Ahem, yes :)

Reply:
fixed